scripts/gasEstimation2.ts

```diff
@@ -114,25 +114,223 @@ async function estimateGasCost() {
     });
     const masterDataTxReceipt = await masterDataUpdate.wait();
 
-    const vaultDataUpdate = await stateRelayerProxy.connect(bot).updateBurnInfo({
-      fee: i,
-      auction: i,
-      payback: i,
-      emission: i,
-      total: i,
-      decimals: i,
+    const burnDataUpdate = await stateRelayerProxy.connect(bot).updateBurnInfo({
+      burnAddress: '8defichainBurnAddressXXXXXXXdRQkSm',
+      amount: '1560417037859082200',
+      tokens: [
+        '7260.95539278@DFI',
+        '17089.00000000@ETH',
```

Comment:
Are you talking about the `tokens` field? If that's the case, if more token get added on DFIchain this will increase and vice versa. If you are talking about individual field, it is highly unlikely that we will burn that amount. However, need to be prepared for that scenario as well.